share/mk/bsd.compat.mk

```diff
@@ -121,8 +120,7 @@ HAS_COMPAT+=64C
 LIB64C_MACHINE=	arm64
 LIB64C_MACHINE_ARCH=	aarch64c
 LIB64CCPUFLAGS=	-target aarch64-unknown-freebsd13.0
-# XXX: Drop -femulated-tls once bsd.cpu.mk no longer passes it
-LIB64CCPUFLAGS+=	-march=morello+c64 -mabi=purecap -femulated-tls
+LIB64CCPUFLAGS+=	-march=morello+c64 -mabi=purecap
```

Comment:
This file needs the varargs changes?

Reply:
bsd.cpu.mk sets it regardless of ABI as it affects both, so it doesn't need repeating here (unless you ever have a vanilla world with lib64c, but we don't support that)

Reply:
Perhaps not since we append these to the CFLAGS from bsd.cpu.mk?

Reply:
Having said that we do redundantly set -march for RISC-V, for example, despite the fact that's not changing between ABIs